### PR TITLE
mantra unsolved

### PR DESCRIPTION
**Pull request by carolineLee123**
@twang 
1.7

### CODE REVIEW

**Comment by twang** [comment]
Thanks @carolineLee123!

@macdestroe:  if you review this late, please reopen the pull request

File: Mantra.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+public static void Mantra(){
```
Comment (macdestroe) [inline Mantra.txt]: This is how you state a method, not how you state a class!

File: Mantra.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+System.out.println("There's one thing every coder must understand:");
```
Comment (macdestroe) [inline Mantra.txt]: You should add in your main method here.

**Comment by twang** [comment]
Got it, thanks @macdestroe!

**Pull request closed without merging**